kythe/cxx/verifier/verifier_unit_test.cc

```diff
@@ -2332,6 +2298,8 @@ target { root:"3" }
   ASSERT_TRUE(v.VerifyAllGoals());
 }
 
+// TODO(zarko): this test fails with the new solver; fix and check the other
+// EqualityConstraint tests.
```

Comment:
nit, optional: keep this TEST_P but use GTEST_SKIP() to document that this is being skipped somewhere other than a comment?